delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1041,20 +1040,42 @@ def updateServiceInDB(service_id_in_db, devApiID, region, statusval) {
 def loadEventsModule() {
 }
 
+	
 def loadServiceInfo() {
 	def token = setCredentials()
 	def url = "$g_base_url/platform/services?domain=$domain&service=$service&isAdmin=true"
 	def curlOutput = sh (script: "curl -H \"Content-Type: application/json\" \
-			-H \"Authorization: $token\" \"$url\" | $jq -r '.data[0] | [.created_by] | join(\",\")'", returnStdout: true)
-	def values = curlOutput.split(',')
-
+			-H \"Authorization: $token\" \"$url\" -w \"httpErrorCode:%{response_code}:\"", returnStdout: true)
+  	     
+    def httpResponse = 0
+    if(curlOutput != null && (curlOutput.indexOf("httpErrorCode") > -1))
+  	{
+     httpResponse = curlOutput.split('httpErrorCode:')[1].split(':')[0]
+  	 echo "httpResponse: $httpResponse"
+  	}
+  
+    if( httpResponse == "200"){
+       echo "httpResponse Success"
+    echo "curlOutput: $curlOutput"
+    	
+	def jsonParser = new groovy.json.JsonSlurper()
+	def resultJson = jsonParser.parseText(curlOutput)
+  	echo "Resultjson: $resultJson"
+	def createby = ''
+  	def values = ''
+  	if(resultJson != null && resultJson.data != null && resultJson.data[0].created_by != null) {
+			//createby = resultJson.data[0].created_by
+      		values = resultJson.data[0].created_by.split(',')
+		}
 	echo "Services : $values"
 
 	if (values[0]) {
 		owner = values[0].replaceAll("[\n\r]", "")
 	}
 }
-
+  else{
+       echo "httpResponse Failure"
```

Comment:
Not too harmful, but you should echo more contextually. Ex: echo "error calling services API for service name: " + servicename + "."
echo "Details: " +  ....

You can choose to provide details or not